test/functional/plugin/lsp_spec.lua

```diff
@@ -6348,7 +6348,7 @@ describe('LSP', function()
         vim.lsp.config('foo', {
           cmd = server.cmd,
           filetypes = { 'foo' },
-          root_dir = function(cb)
+          root_dir = function(_bufnr, cb)
```

Comment:
Can we add a test where we use the buffer number?

Reply:
Do you have any suggestions? Do you mean just add a line that uses the buffer number?

Reply:
Yeah? What's the output of `nvim_buf_get_name` when called from the tests?

Feel free to not add the test if you find the check not useful.